### ISSUE TITLE
Images do not show up in Foxx Micro-service markdown "Readme" view

### ISSUE THREAD

**Issue by aherreraGH** [arangodb/arangodb#5780]
## My Environment

* __ArangoDB Version__:        3.3.11
* __Storage Engine__:             N/A
* __Deployment Mode__:       Single Server
* __Configuration__:               default install, foxx microservice
* __Infrastructure__:               own 
* __Operating System__:        Windows 10
* __Used Package__:              Docker - official Docker library

## Component, Query & Data

__Affected feature__:
Foxx/ArangoDB web interface

__Dataset__:
None

## Steps to reproduce

1.  Create a README.md file, set it at the root level of the Foxx service directory
2. Add an image to the same directory as the README.md file. Or create a directory and add the images in there.
3. Add a reference to the image in README.md: ![API URL](api_url_2.png)
2.  Upload Foxx service
3.  Review Services within the ArangoDB web interface, and view the "home" page for the new Foxx service.

__Problem__:
Images show up as broken links/images

**Side Note:** This can go for imported markdown files as well.  It would be nice to breakdown the markdown files into several small files. 

__Expected result__:
Images should be visible.

**Comment by pluma** [comment]
There's no easy way to handle serving documentation assets in the web interface right now. The readme is simply parsed into HTML and rendered but there's no logic for providing images or multi-document links.

We might want to provide a way to specify a "docs" folder for the service and serve that from the web interface.

**Comment by aherreraGH** [comment]
okay. Thank you. 
Would it be something like:
```
(main directory0
README.md
docs
   |__ child1.md
   |__ image1.png
```
then README.md and child1.md will know to find images in something like /docs/*.png ?

or
```
(main directory0
docs
   |__ README.md
   |__ child1.md
   |__ image1.png
```

Just making sure I'm following along with what you may look to implement. Thank you.

**Comment by hallicarnassus** [comment]
There is a way to do this using the current version.

1.  In your manifest.js file, define a location for the image (relative to your index.js file):
`"files":{
  "/images": "/images"
}`

2. Copy the images to this folder
3. You can serve images using direct links to the image required, eg:

[readme.md] (located in the root of the Foxx Service)

```
 **FoxxMicroservice ReadMe**
 A simple example of how to serve an image to the readme.md page of your FoxxMicroservice
 
 ![Alt text](http://localhost:8529/_db/dbName/FoxxMicroserviceName/images/image.png?raw=true "image description")
```

**Comment by pluma** [comment]
@hallicarnassus that's not really the same thing, though. You're literally serving the files as part of your service whereas the README is served by ArangoDB itself. Your workaround also requires the user to know the database and path the service is going to be mounted in.

**Comment by hallicarnassus** [comment]
@pluma Agreed, it does work though...  shouldn't the admin/dev already have this apriori knowledge anyway?

**Comment by pluma** [comment]
@hallicarnassus hardcoding database and mount paths into the service is a pretty bad idea. These should be considered implementation details. You might want the same service installed in multiple databases, for example.

**Comment by aherreraGH** [comment]
@pluma - I am looking to have this installed in multiple databases, definitely.
